scripts/cangen/cangen/traffic.py

```diff
@@ -0,0 +1,48 @@
+"""
+Author: Manush Patel
+Date: 2024-11-24
+https://macformula.github.io/racecar/firmware/can-traffic/
+"""
+
+from dataclasses import dataclass
+
+
+@dataclass
+class CANMessage:
+    data_length: int
+    frequency: int
+
+    @property
+    def bit_count(self):
+        fixed_length = 44  # fixed length of CAN frame
+        variable_length = self.data_length * 8.0  # accounted for bit stuffing
+        total_frame_length = variable_length + fixed_length
+        return total_frame_length
+
+
+def calculate_bus_load(messages: list[CANMessage], can_speed: int) -> float:
+    """Calculates the bus load percentage based on CAN messages and bus speed
+
+    Parameters
+    ----------
+    messages : list[CANMessage]
+        A list of CANMessage objects, each containing `bit_count` and `frequency` attributes
```

Comment:
This is not a helpful comment. You can remove this line. The previous line already describes that this is "a list of CANMessage objects." Also, you do not want to describe the internal attributes of that object. The purpose of an object is to encapsulate data, so this function should not need to know how the class works.